docs/docs/how-tos/domain-registry.md

```diff
@@ -6,12 +6,13 @@ description: Setting a custom DNS Domain Registry
 
 # Nebari domain registry setup
 
```

Comment:
### What is a DNS?

The Domain Name System (DNS) turns domain names into IP addresses, which browsers use to load internet pages.
Every device connected to the internet has its IP address, which other devices use to locate the device. 
DNS name servers are used to locate the IP address of a domain name using words or instead of direct IP addresses. 

### Setting up a DNS 

During deployment, Qhub will generate an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) IP for connection with the Kubernetes cluster and all related services that Qhub runs. If not automatically handled, Qhub will request the user to generate the necessary [DNS records](https://www.cloudflare.com/learning/dns/dns-records/) and update the domain within the newly created IP:

```
Take IP Address 12.312.312.312 and update DNS to point to "your.domain" [Press Enter when Complete]
```

Once the IP is generated, you will need to grab it and create the necessary records within the DNS provider of your choice.